clifford/_multivector.py

```diff
@@ -745,6 +749,52 @@ def normalInv(self, check=True) -> 'MultiVector':
         """
         return self._pick_inv(fallback=False if check else None)
 
+    def hitzer_inverse(self):
+        """
+        Performs the inversion operation as described in the paper:
+        Multivector and multivector matrix inverses in real Clifford algebras
+        Eckhard Hitzer, Stephen Sangwine (2017)
```

Comment:
I think we should get the Sphinx citation thing set up so that we can start making these things easier to click through. I'll maybe set that up today.

Reply:
xref gh-339

Reply:
Now that this is done, can you add to the `.bib` file?

Reply:
Done :)